.github/workflows/zowe-exlorer-ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Zowe Explorer CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      # Continue to run tests on the other systems if one fails
+      fail-fast: false
+      matrix:
+        node-version: [10.x, 12.x]
+        # order operating systems from best to worst
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - run: npm install
+
+    - name: Audit from Windows / Node 12.x
+      run: npm audit --audit-level=moderate
+      if: matrix.os == 'windows-latest' && matrix.node-version == '12.x'
```

Comment:
What is the purpose of this?

Reply:
in retrospect, probably no purpose 😄 

I'll remove it - it should be no problem to audit for all combinations

Reply:
this check  is removed